elements/rh-spinner/rh-spinner.ts

```diff
@@ -37,16 +43,19 @@ export class RhSpinner extends LitElement {
    @colorContextConsumer()
    @property({ reflect: true }) on?: ColorTheme;
 
+   /** Preset sizes for the spinner */
+  @property({ reflect: true }) size: SpinnerSize = 'lg';
+
 
-   render() {
-     return html`
+  render() {
+    return html`
         <svg role="progressbar" viewBox="0 0 100 100" aria-live="polite">
           <circle class="track" cx="50" cy="50" r="40" fill="none" vector-effect="non-scaling-stroke" />
           <circle class="dash" cx="50" cy="50" r="40" fill="none" vector-effect="non-scaling-stroke" />
         </svg>
-        <p><slot></slot></p>
```

Comment:
did you want to always show the slotted text?

Reply:
@markcaron, what do you think?